test/test.dart

```diff
@@ -561,6 +566,7 @@ Future<void> baasTest(
   AppNames appName = AppNames.flexible,
   dynamic skip,
 }) async {
+  if (BaasClient.initializationError != null) throw BaasClient.initializationError!;
```

Comment:
small one but... can we wrap this in brackets?
```suggestion
  if (BaasClient.initializationError != null){
   throw BaasClient.initializationError!;
   }
```